docs/site/Interceptors.md

```diff
@@ -0,0 +1,328 @@
+---
+lang: en
+title: 'Interceptors'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Interceptors.html
+---
+
+## Overview
+
+Interceptors are reusable functions to provide aspect-oriented logic around
+method invocations. There are many use cases for interceptors, such as:
+
+- Add extra logic before / after method invocation, for example, logging or
+  measuring method invocations.
+- Validate/transform arguments
+- Validate/transform return values
+- Catch/transform errors, for example, normalize error objects
+- Override the method invocation, for example, return from cache
+
+## Interceptor functions
+
+The interceptor function has the following signature:
+
+```ts
+export interface Interceptor {
+  <T>(
+    context: InvocationContext,
+    next: () => ValueOrPromise<T>,
+  ): ValueOrPromise<T>;
+}
+```
+
+An interceptor is responsible for calling `next()` if it wants to proceed with
+next interceptor or the target method invocation.
```

Comment:
I am afraid this sentence is not enough to explain `next`. I was confused about this concept until I read example interceptors below, at which point I started to understand that you are using `next` the same way as Koa does.

Please add more details here to better explain the concept of `next`, how can interceptors use it to run code after the target methods has finished, etc.